pkgs/pboy.yaml

```diff
@@ -0,0 +1,31 @@
+title: pboy
+tagline: Console-based PDF management utility.
+about: |
+  Paperboy (pboy) is lightweight PDF management tool for the command-line.
+  It helps to rename and move files to a specified folder, and it even 
+  gives some filename suggestions by looking at the content and the PDF
+  metadata.
+
+git_user: 2mol
+git_repo: pboy
+info_url: https://github.com/[GIT_USER]/[GIT_REPO]
+releases_url: https://github.com/[GIT_USER]/[GIT_REPO]/releases/
+source_url: https://github.com/[GIT_USER]/[GIT_REPO]/
+
+base_download_url: https://github.com/[GIT_USER]/[GIT_REPO]/releases/download/[VER]/
+filename_format: pboy-[OS]
+version_format:
+latest_strategy: github-release # uses most recent github release tag to get the latest version number
+
+is_binary: false
+
+os_map:
+  linux:
+    name: linux
+    ext: tar.gz
+  macos: 
+    name: osx
+    ext: tar.gz
+  # win: Unsupported
+arch_map:
+  amd64: 
```

Comment:
What are you mapping the "amd64" to?
If the arch string looks the same ("amd64"), then have "amd64: amd64"

Reply:
Also, it looks like you've got no bin_paths for these. Make sure to add those.